src/Database/Relations/BelongsToMany.php

```diff
@@ -100,6 +100,110 @@ public function create(array $attributes = [], array $pivotData = [], $sessionKe
         return $model;
     }
 
+    /**
+     * @param mixed $id
+     * @param array $attributes
+     * @param bool  $touch
+     */
+    public function attach($id, array $attributes = [], $touch = true)
+    {
+        $insertData = $this->formatAttachRecords($this->parseIds($id), $attributes);
```

Comment:
This is fine as just `$data`

Reply:
I do not agree, "$data" variable name is too abstract.